db/migrate/20200310034130_create_jobs.rb

```diff
@@ -0,0 +1,16 @@
+class CreateJobs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :jobs do |t|
+      t.string :job_title
+      t.text :job_description
+      t.boolean :active
+      t.references :company, foreign_key: true
+      t.string :location
+      t.boolean :remote
+      t.date :start_date
```

Comment:
didn't we say this should be a text field where companies can elaborate instead of having to pick one date specifically?

Reply:
yep, cool. Text field it is.